common/app/views/support/DropdownMenus.scala

```diff
@@ -39,6 +39,7 @@ object DropdownMenus {
       href = Some(s"${Configuration.id.mmaUrl}/account-settings"),
       linkName = Some("settings"),
       label = "Settings",
+      classList = List("js-user-account-dropdown-menu-settings-item"),
```

Comment:
the menu is server-side rendered, and we need to find the `Settings` item client-side to add notifications + the dot